src/gui/plugins/component_inspector/JointType.qml

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtQuick.Controls 1.4
+import QtQuick.Controls 2.2
+import QtQuick.Controls.Material 2.1
+import QtQuick.Layouts 1.3
+import QtQuick.Controls.Styles 1.4
+import "qrc:/ComponentInspector"
+import "qrc:/qml"
+
+// Item displaying joint type information.
+Rectangle {
+  id: jointTypeComponent
+  height: jointType.height
+  width: componentInspector.width
+  color: index % 2 == 0 ? lightGrey : darkGrey
+
+  // Left indentation
+  property int indentation: 10
```

Comment:
This component looks more indented than the rest

![image](https://user-images.githubusercontent.com/5751272/144685358-001c8191-0259-4539-85a5-02296b6153c6.png)

Reply:
Try this: cc3ee2b